djangocms_link/forms.py

```diff
@@ -1,34 +1,38 @@
 # -*- coding: utf-8 -*-
-from __future__ import absolute_import, print_function, unicode_literals
-
 from django.forms import ValidationError
 from django.forms.models import ModelForm
 from django.forms.widgets import Media
 from django.utils.translation import ugettext_lazy as _
 
 from djangocms_attributes_field.widgets import AttributesWidget
 
-from djangocms_link.models import Link
+from .models import Link
 
 
 class LinkForm(ModelForm):
```

Comment:
instead of having two layers of conditionals, one here and one in `fields.py`, I would set `PageSearchField` to be the name for both the `select2` field and the page field from cms and so if you use `PageSearchField` from `fields.py` it will always point to the correct one.